### DISCUSSION TITLE
My school deletes cookies and cache when browser/chromebook restarts.

### DISCUSSION THREAD

**Discussion by 1-Mythical** [3kh0/ext-remover#509] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
How do I save cookies/cache/local storage into a file or something to preserve it for next restart?

**Comment by qDread-dev** [comment, marked as answer]
Just dont restart 🤷

**Reply by BroJac5246** [reply]
Some schools force them to restart when closed, so that might not be possible.

**Reply by 1-Mythical** [reply]
My school has a force restart notification that appears every Wednesday at 10. RIP Retro Bowl data 😞

**Reply by qDread-dev** [reply]
RIP
